parser/Builder.cc

```diff
@@ -832,6 +885,13 @@ class Builder::Impl {
         return make_unique<NthRef>(tokLoc(tok), atoi(tok->string().c_str()));
     }
 
+    unique_ptr<Node> numparams(size_t max_numparam) {
+        // At this point, we might not have a node to get the location so we use a dummy one for now,
+        // errors will later be rattached to the block itself.
+        auto loc = core::LocOffsets();
```

Comment:
I don't see this used anywhere else in the codebase. Maybe you want this:

```suggestion
        auto loc = core::LocOffsets::none();
```

Reply:
We can't use `none` as we have a sanity check at the and of the parsing phase that `Exception::raise(): parser/Node.h:11 enforced condition loc.exists() has failed: Location of parser node is none` (https://github.com/sorbet/sorbet/blob/master/parser/Node.h#L11). That's why I was using an empty one instead (implicit default constructor).

I added a comment to clarify all of this.

I can create a more explicit `dummy()` constructor if you want.

Reply:
In that case you can write `LocOffsets(0, 0)` to make it clear that the intention was a zero-width loc.